### PR TITLE
npm-why: add German translation

### PR DESCRIPTION
**Pull request by schwarmco**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.de/common/npm-why.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# npm-why
+
+> Identifiziert, warum ein NPM Paket installiert ist.
+> Weitere Informationen: <https://www.npmjs.com/package/npm-why>.
+
+- Zeige, warum ein NPM Paket installiert ist:
```
Comment (navarroaxel) [inline pages.de/common/npm-why.md:6]: Do the `npm` brand guidelines tell to write it in lowercase? :thinking: 
```suggestion
- Zeige, warum ein npm Paket installiert ist:
```

File: pages.de/common/npm-why.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# npm-why
+
+> Identifiziert, warum ein NPM Paket installiert ist.
```
Comment (schwarmco) [inline pages.de/common/npm-why.md:3]:
```suggestion
> Identifiziert, warum ein npm Paket installiert ist.
```

File: pages.de/common/npm-why.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+> Weitere Informationen: <https://www.npmjs.com/package/npm-why>.
+
+- Zeige, warum ein NPM Paket installiert ist:
+
+`npm-why {{package_name}}`
```
Comment (schwarmco) [inline pages.de/common/npm-why.md:8]:
```suggestion
`npm-why {{paket_name}}`
```

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @schwarmco

**Pull request merged**